change-beta/@azure-communication-react-df6120b2-05e7-4366-b01f-5c914db4a2a2.json

```diff
@@ -0,0 +1,9 @@
+{
+  "type": "none",
+  "area": "fix",
+  "workstream": "",
+  "comment": "Fix sample appl issue where after selecting pstn call then switching to start group call, the calls fails trying to make a PSTN call",
```

Comment:
```suggestion
  "comment": "Fix sample app issue where after selecting pstn call then switching to start group call, the calls fails trying to make a PSTN call",
```

Reply:
I totally agree with this spelling error fix, but to avoid the (albiet minute) time it takes to hop on a branch and fix this, given this changelog is of type `none` I am going to leave as is